mei40To50/mei40To50.xsl

```diff
@@ -231,19 +306,114 @@
+            <xsl:choose>
+                <xsl:when test="$value.old = 'false'">none</xsl:when>
+                <xsl:otherwise>before</xsl:otherwise>
+            </xsl:choose>
+        </xsl:variable>
+        <xsl:attribute name="keysig.cancelaccid" select="$value.new"/>
+        <xsl:if test="$verbose">
+            <xsl:message>
+                <xsl:value-of select="'Changing @keysig.showchange to @keysig.cancelaccid on ' || local-name(parent::mei:*) || ' ' || ancestor-or-self::mei:*[@xml:id][1]/@xml:id || '. '"/>
+                <xsl:text>Please note: converting value from </xsl:text>
+                <xsl:value-of select="$value.old"/><xsl:text> to </xsl:text><xsl:value-of select="$value.new"/><xsl:text>. </xsl:text>
+                <xsl:if test="$value.old = 'true'">There are several alternatives for encoding the style of MEI 4 @keysig.showchange='true' in MEI 5.0 (before, after, before-bar), defaulting to 'before'.</xsl:if>
+            </xsl:message>
+        </xsl:if>
+    </xsl:template>
+    
+    <xd:doc>
+        <xd:desc>
+            <xd:p>Replace @sig.showchange with @cancelaccid.</xd:p>
+        </xd:desc>
+    </xd:doc>
+    <xsl:template match="@sig.showchange">
+        <xsl:variable name="value.old" select="."/>
+        <xsl:variable name="value.new">
+            <xsl:choose>
+                <xsl:when test="$value.old = 'false'">none</xsl:when>
+                <xsl:otherwise>before</xsl:otherwise>
+            </xsl:choose>
+        </xsl:variable>
+        <xsl:attribute name="cancelaccid" select="$value.new"/>
+        <xsl:if test="$verbose">
+            <xsl:message>
+                <xsl:value-of select="'Changing @sig.showchange to @cancelaccid on ' || local-name(parent::mei:*) || ' ' || ancestor-or-self::mei:*[@xml:id][1]/@xml:id || '. '"/>
+                <xsl:text>Please note: converting value from </xsl:text>
+                <xsl:value-of select="$value.old"/><xsl:text> to </xsl:text><xsl:value-of select="$value.new"/><xsl:text>. </xsl:text>
+                <xsl:if test="$value.old = 'true'">There are several alternatives for encoding the style of MEI 4 @sig.showchange='true' in MEI 5.0 (before, after, before-bar), defaulting to 'before'.</xsl:if>
+            </xsl:message>
+        </xsl:if>
+    </xsl:template>
+    
+    <xd:doc>
+        <xd:desc>Resolve changes in @meter.form, moving @meter.form="invis" to @meter.visible</xd:desc>
+    </xd:doc>
+    <xsl:template match="@meter.form">
+        <xsl:choose>
+            <xsl:when test=". = 'invis'">
+                <xsl:attribute name="meter.visible">false</xsl:attribute>
+                <xsl:if test="$verbose">
+                    <xsl:message>Changing @meter with value "invis" to @meter.visible with value "false"</xsl:message>
+                </xsl:if>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:copy-of select="."/>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+    
     <xd:doc>
```

Comment:
```suggestion
    <xd:doc>
        <xd:desc>Resolve changes in meterSig/@form, moving @form="invis" to @visible="false"</xd:desc>
    </xd:doc>
    <xsl:template match="mei:meterSig/@form">
        <xsl:choose>
            <xsl:when test=". = 'invis'">
                <xsl:attribute name="visible">false</xsl:attribute>
                <xsl:if test="$verbose">
                    <xsl:message select="'Changing @form with value `invis` to @visible with value `false` on meterSig ' || ancestor-or-self::mei:*[@xml:id][1]/@xml:id"/>
                </xsl:if>
            </xsl:when>
            <xsl:otherwise>
                <xsl:copy-of select="."/>
            </xsl:otherwise>
        </xsl:choose>
    </xsl:template>
    
    <xd:doc>
```

Reply:
@bwbohl Please double check